packages/backend-data/package.json

```diff
@@ -29,7 +29,7 @@
   "dependencies": {
     "@aws-amplify/backend-output-storage": "0.2.4",
     "@aws-amplify/backend-output-schemas": "^0.4.0",
-    "@aws-amplify/data-construct": "^1.4.1",
+    "@aws-amplify/data-construct": "1.4.1",
```

Comment:
what motivates this change?
wouldn't customers have option to pin it with package manager if necessary ?

Reply:
Was thinking about this after posting, and we can also do things like update `latest` tag if a regression is found urgently (we need to get into that habit anyway). We should probably build those muscles now, not at GA.

This was discussed in standup today to bring stability during preview launch window, circling back w/ the team.

Reply:
Yeah, after reflecting on it a bit. I think not doing this is probably better for long-term health.